### ISSUE TITLE
Detect device & send Android users from browser to play store

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/webmaker-desktop#62]
Android users who open a shared project link should be sent to the app in the google play store.

Reference diagram by @flukeout https://github.com/mozilla/webmaker-android/issues/1616#issuecomment-101424066

**Comment by sanjaygouri** [comment]
Hello @xmatthewx i want to work on this . So what we gonna do is whenever a user opens  link[1] he has to redirected to android play store if he is using android other wise it has to open in the browser itself right?

1.https://beta.webmaker.org/

**Comment by xmatthewx** [comment]
Thanks @sanjaygouri 

A few notes: 
- beta.webmaker.org is a temporary site. It will grow and replace webmaker.org when it's ready. 
- Android users without the app should forward to the app in the play store like you mentioned. We will redirect project URLs, but not informational pages such as webmaker.org/terms.

@thisandagain - Do we want to forward landing page traffic? Or just project URLs?

**Comment by xmatthewx** [comment]
@sanjaygouri - Talking with the team, it seems like we need a few more things put in place before we can wrangle this task.

**Comment by xmatthewx** [comment]
This is one thing we didn't include in our triage today.

**Comment by xmatthewx** [comment]
@k88hudson - can you summarize the technical constraints we should update design around? Sounds like we should load project in browser and then provide a button to open in the app.

**Comment by k88hudson** [comment]
In order to open a link inside the app, the action must be initiated by a _user gesture_, such as a click or a tap.

**Comment by xmatthewx** [comment]
@vazquez - you can post question here about we should/shouldn't combine "open in app" with "get the app" CTA.

**Comment by vazquez** [comment]
@k88hudson - Do we know if a person has the Webmaker app installed? 

If not, should a generic "open this project in app" also serve as Get The App button?

**Comment by vazquez** [comment]
I just spoke IRL with @k88hudson.

We can't know if a person has the Webmaker app installed.

Therefore, we need to include a generic "Open in Webmaker" notice and button.

Android is smart though,
- If the user has Webmaker App installed, the App will open when user taps the CTA.
- If the user doesn't have Webmaker App installed, Android directs the user to the Play Store.

**Comment by vazquez** [comment]
@xmatthewx :arrow_up:

**Comment by xmatthewx** [comment]
Sounds like a solid plan. Generic button is step one. Later, we should probably detect desktop and serve up our branded play store button. So, don't drop that artwork in the shredder.

**Comment by xmatthewx** [comment]
Closed by #322. Tracking device detection here https://github.com/mozilla/webmaker-core/issues/360
